RetailCoder.VBE/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -361,9 +359,9 @@ private void ExecuteQuickFixInModuleCommand(object parameter)
                 .ToList();
 
             var items = filteredResults
-                .Where(result => !(result is AggregateInspectionResult))
+                .Where(result => result.GetType().Name != "AggregateInspectionResult")
```

Comment:
First of all, use `nameof`--that'll fix the type-renaming issue. Second, why not use the `is`?